departments/development/rfp/README.md

```diff
@@ -4,13 +4,20 @@ The purpose of this folder is to store all the RFPs (Requests for Proposals) ini
 
 ## RFP folder naming convention
 
-The folder created for each RFP should follow the convention below::
+Each RFP should be stored in a folder named with the following convention:
 
-RFP-[RFP_TITLE]-[STATUS]
+`[SEQUENTIAL_NUMBER]-[RFP_TITLE]`
 
-Example: "RFP-Chain Upgrade Support-Open"
+Example:
+`001-Chain_Upgrade_Support`
 
+- `SEQUENTIAL_NUMBER`: A unique number assigned to each RFP in sequential order, starting from 001.
```

Comment:
Here we are saying that the RFP ID (integer) is not global but Department-specific.
This can be problematic for applicants or the general public, and in case we want Council-level RFPs.

Reply:
this was discussed in the dev dept meeting. 
since sequential assignments are manual - it would be easier to publish RFPs publicly using a prefix like:

DEV-001
COMM-001
OPS-001

rather than attempting to make the RFP number sequential across all departments